homeassistant/components/simplefin/const.py

```diff
@@ -9,4 +9,4 @@
 
 LOGGER = logging.getLogger(__package__)
 
-CONF_ACCESS_URL = "access_url"
+CONF_ACCESS_URL: str = "access_url"
```

Comment:
```suggestion
CONF_ACCESS_URL = "access_url"
```

Should not be needed and detected automatically